docs/estela/api/endpointsmd.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Enpoints
+parent: API
+grand_parent: estela
+---
+
+# Endpoints
+estela offers different endpoints to interact with the users, the web and queueing modules. For example:
```

Comment:
This text implies that `the users` is a module, to avoid misunderstandings, please change it to _with the users, as well as the web and queueing modules_. Or some similar text that shows the difference.

Reply:
you are right